### PR TITLE
oneDNN v3.11 release notes

### PR DESCRIPTION
**Pull request by vgvozdeva**
This PR includes a release notes draft based on the information from the PRs for the contributors to review. Your additions and corrections are highly appreciated.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* Improved `fp32` matmul performance with `fp4` compressed weights.
+* Improved `fp32` matmul performance for cases when one of the tensors has a trivial dimension on processors with Intel AVX-512 instruction set support.
+
+## Intel Graphics Products
+* Improved `fp16`/`bf16` matmul performance for large tensor cases on Intel Arc graphics for Intel Core Ultra processor series 3 (formerly Panther Lake).
+* Improved matmul performance for cases with 4-byte alignment on Intel GPUs based on Xe2 architecture.
+* Improved performance of `fp16`/`bf16` matmul with `mxfp4` weights.
+* Improved convolution performance with host-side scalar scales and zero points.
+
+## AArch64-based Processors
```
Comment (vgvozdeva) [inline RELEASE_NOTES.md]: @Sqvid @Ryo-not-rio could you please help summarizing AArch64 improvements?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Performance Optimizations
+## Intel Architecture Processors
```
Comment (vgvozdeva) [inline RELEASE_NOTES.md]: @densamoilov @ankalinin could you please review and update CPU part if required?

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* Improved `fp32` matmul performance with `fp4` compressed weights.
+* Improved `fp32` matmul performance for cases when one of the tensors has a trivial dimension on processors with Intel AVX-512 instruction set support.
+
+## Intel Graphics Products
```
Comment (vgvozdeva) [inline RELEASE_NOTES.md]: @karturov @echeresh could you please review and update section if required?

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+* Improved `fp16`/`bf16` matmul performance for large tensor cases on Intel Arc graphics for Intel Core Ultra processor series 3 (formerly Panther Lake).
+* Improved matmul performance for cases with 4-byte alignment on Intel GPUs based on Xe2 architecture.
+* Improved performance of `fp16`/`bf16` matmul with `mxfp4` weights.
+* Improved convolution performance with host-side scalar scales and zero points.
+
+## AArch64-based Processors
+TBD
+
```
Comment (vpirogov) [inline RELEASE_NOTES.md:22]: @uxlfoundation/onednn-cpu-rv64, could you please summarize RISC-V specific improvements optimizations included in this release? Release notes are structured into three major sections:
* "Performance optimizations" section highlights functionality that works faster in the new release
* "Functionality" section highlights functionality that did not work before (i.e. new public API introduced or existing API that returned `unimplemented` before)
* "Usability" section covers all other improvements


```suggestion
## RISC-V based processors
*
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+* Introduced destination tensor [dynamic quantization] in matmul primitive following Open Compute Microscaling (MX) formats specification. See [MXFP8 matmul tutorial] for quick introduction into MX-capabilities in oneDNN.
+* Introduced support for NVFP4 quantization scheme. The changes include support for `fp8_e4m3` grouped scales and dynamic quantization support for destination tensor with NVFP4-specific formula for scales computation.
+* Introduced support for dropout as a primitive attribute for matmul, softmax and eltwise primitives.
+
+[dynamic quantization]: https://uxlfoundation.github.io/oneDNN/v3.11/dev_guide_attributes_quantization.html#dynamic-quantization
+[MXFP8 matmul tutorial]: https://uxlfoundation.github.io/oneDNN/v3.11/page_mxfp_matmul_cpp.html#doxid-mxfp-matmul-cpp
+
+## Graph API 
+* Introduced support for [RMS Normalization] operation. 
+* Introduced support for output gradient of attention mask for SDPA training.
```
Comment (ElaineBao) [inline RELEASE_NOTES.md]:
```suggestion
* Introduced support for output gradient of attention mask for SDPA and GQA training.
```

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+[RMS Normalization]: https://uxlfoundation.github.io/oneDNN/v3.11/dev_guide_op_rmsnorm.html
+
+## Intel Graphics Products
+* Introduced support for convolution with `u8` weights.
+* Introduced support for 2D grouped scales in `fp8` matmul.
+
+## Intel Architecture Processors
+* Introduced support for different data types of source and destination in pooling forward propagation.
+
+# Usability
```
Comment (TaoLv) [inline RELEASE_NOTES.md:58]: @ElaineBao  Do we need a sentence here for graph dump which now is enabled by default?

**Review by TaoLv — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+* Extended [quantization attributes] documentation to cover all quantization schemes supported by the library.
+* Added [matmul fp8 quantization] example demonstrating use of matmul primitive with `fp8` source, destination, and weights.
+* Extended oneDNN [threadpool runtime] with an option to support asynchronous execution and updated all CPU implementations accordingly. This extension makes oneDNN compatible with OpenXLA "thunk" runtime.
+* Extended information about primitive execution available in VTune(TM) Profiler with the same level of detail as reported by oneDNN [verbose mode]. This feature requires VTune Profiler 2025.7 or later.
+* Introduced [`ONEDNN_SAFE_RBP`] build knob that instructs x64 implementations to preserve value of `rbp` register for tools that rely on stack unwinding. This option may have visible performance impact on some workloads.
+* Removed build time dependency on OpenCL runtime in SYCL build configuration.
+
```
Comment (ElaineBao) [inline RELEASE_NOTES.md:74]:
```suggestion
`ONEDNN_ENABLE_GRAPH_DUMP` build knob is enabled by default.

```

File: RELEASE_NOTES.md (hunk 6 above)
Comment (ElaineBao) [reply RELEASE_NOTES.md:58]: thanks for reminding, added.

File: RELEASE_NOTES.md (hunk 4 above)
Comment (zhangfeiv0) [reply RELEASE_NOTES.md:22]: ## RISC-V based processors
* Improved eltwise primitive performance.
* Improved binary primitive performance.
* Improved `f32` GEMM performance.
* Improved `f32` convolution primitive performance.
* Improved `f32` matmul primitive performance
* Improved `f32` softmax primitive performance
* Improved `f32` batch normalization primitive performance.
* Improved `f32` group normalization primitive performance.
* Improved `f32` layer normalization primitive performance.
* Improved `f32` and `fp16` pooling primitive performance.
* Improved `f32` inner product primitive performance.
* Improved reorder(`fp32` to `u8`) primitive performance.

Comment (zhangfeiv0) [inline RELEASE_NOTES.md:22]: for more comments. @zhangjian29

Comment (vgvozdeva) [reply RELEASE_NOTES.md]: Added in last commit

File: RELEASE_NOTES.md (hunk 1 above)
Comment (Sqvid) [reply RELEASE_NOTES.md]: Done.

**Review by vpirogov — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 8]
```diff
@@ -4,7 +4,7 @@
 * Improved `fp32` matmul performance for cases when one of the tensors has a trivial dimension on processors with Intel AVX-512 instruction set support.
 
 ## Intel Graphics
-* Improved `fp16`/`bf16` matmul performance for large tensor cases on Intel Graphics for Intel Core Ultra Series 3 processor (formerly Panther Lake).
+* Improved `fp16`/`bf16` matmul performance for large tensor cases on Intel Graphics for Intel Core Ultra processor Series 3 (formerly Panther Lake).
```
Comment (vpirogov) [inline RELEASE_NOTES.md:7]: Previous variant was just fine. Check this out:
https://www.intel.com/content/www/us/en/products/details/processors/core-ultra.html

**Review by echeresh — APPROVED** [review]
Slightly extended the GPU sections, looks good to me now.
